src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,14 @@
+package wolox.training.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import wolox.training.models.Book;
+/**
+ * Repository Interface for {@link Book} class that extends {@link JpaRepository}
+ *
+ * @author federicodinucci
+**/
+public interface BookRepository extends JpaRepository<Book, Long> {
+
+    Book findByAuthor(String author);
```

Comment:
You can wrap this return value (Book) in an optional:

```
Optional<Book> findByAuthor(String author);
```

You can read more about Java Optionals [here](https://www.baeldung.com/java-optional)